app/views/passengers/index.haml

```diff
@@ -6,20 +6,15 @@
 = form_tag passengers_path, method: :get do
   %p
   = label_tag 'Filter by Type:'
-  Permanent
-  = radio_button_tag :filter, 'permanent'
-  .checkbox-inline
-  Temporary
-  = radio_button_tag :filter, 'temporary'
-  .checkbox-inline
-  Inactive
-  = radio_button_tag :filter, 'inactive'
-  .checkbox-inline
-  Only Active
-  = radio_button_tag :filter, 'active'
-  .checkbox-inline
-  Show All
-  = radio_button_tag :filter, 'all'
+  .passenger-filters
+    Permanent Only
+    = radio_button_tag :filter, 'permanent', @permanent
+    Temporary Only
+    = radio_button_tag :filter, 'temporary', @temporary
+    All
+    = radio_button_tag :filter, '', !(@permanent || @temporary)
```

Comment:
Open to making another instance variable out of this with an `else` in the first conditional in the controller.